applications/ray/kuberay-tpu-webhook/main.go

```diff
@@ -77,7 +77,7 @@ func containerRequestingTPUs(containers ...corev1.Container) bool {
 	return false
 }
 
```

Comment:
Are `clusterName`, `namespace`, and `groupName` only used for logging? It seems a bit odd to me that these are required parameters for a fairly deterministic function.

Reply:
Yeah that's the case for `getNumTPUHostsFromTopology`, the logic behind adding them was so that we could have klog Error and Info statements that would be traceable to the right Group, Namespace, and RayCluster if it failed in this function.

Reply:
I can move the log statements somewhere else, but we should always have `clusterName`, `namespace`, and `groupName` (if not it'd return an error before `getNumTPUHostsFromTopology` so it felt safe to pass them to the function.